src/libcore/hash/mod.rs

```diff
@@ -664,13 +664,39 @@ mod impls {
         }
     }
 
+    #[unstable(feature = "unsized_pointer_hash", issue = "0")] // TODO
```

Comment:
I thought `impl` are insta-stable? 

Also, please change the TODO to FIXME to pass `tidy` check.

```
[00:03:26] tidy error: /checkout/src/libcore/hash/mod.rs:667: TODO is deprecated; use FIXME
[00:03:26] tidy error: /checkout/src/libcore/hash/mod.rs:687: TODO is deprecated; use FIXME
```

Reply:
Fixed.